virt/live_migration/virt-about-live-migration.adoc

```diff
@@ -0,0 +1,42 @@
+:_content-type: ASSEMBLY
+[id="virt-about-live-migration"]
+= About live migration
+include::_attributes/common-attributes.adoc[]
+:context: virt-about-live-migration
+
+toc::[]
+
+Live migration is the process of moving a running virtual machine (VM) to another node in the cluster without interrupting the virtual workload or access. By default, live migration traffic is encrypted using Transport Layer Security (TLS).
+
+You can manually xref:../../virt/live_migration/virt-initiating-live-migration.adoc#virt-initiating-live-migration[initate live migration] for a specific VM.
+
+You can monitor the progress of all live migrations on the xref:../../virt/getting_started/virt-web-console-overview.adoc#overview-migrations_virt-web-console-overview[*Overview* -> *Migrations* tab] in the web console.
+
+You can view the metrics of a VM migration on the xref:../../virt/getting_started/virt-web-console-overview.adoc#virtualmachine-details-metrics_virt-web-console-overview[*VirtualMachine details* -> *Metrics* tab] in the web console.
+
+.Prerequisites
+
+* The cluster must have shared storage with `ReadWriteMany` (RWX) access mode.
+* The cluster must have sufficient RAM and network bandwidth.
++
+[NOTE]
+====
+You must ensure that there is enough memory request capacity in the cluster to support node drains that result in live migrations. You can determine the approximate required spare memory by using the following calculation:
```

Comment:
current page looks a bit confusing - it is about Live Migration as a process, but here and below you are saying about Draining nodes and Eviction Strategy which are mostly related to the Node Maintenance